src/python/pants/backend/shell/target_types.py

```diff
@@ -269,8 +269,8 @@ class ShellCommandOutputsField(StringSequenceField):
     )
 
 
-class ShellCommandDependenciesField(Dependencies):
-    pass
+class ShellCommandDependenciesField(ShellDependenciesField):
```

Comment:
Not strictly necessary, since these have no sources, and so don't participate in dep inference, but good for uniformity (and to support transitive excludes, should those be necessary for some reason).

Reply:
FWIW I thought dep inference now uses a field set, and therefore doesn't need a source field?

Reply:
Oh, should that line be updated to instead look for some other Shell sentinel field?

Reply:
Hm, this is also suspect? https://github.com/pantsbuild/pants/blob/7f98eada6ea15f264226c47b2d1da0bd876cd74f/src/python/pants/backend/shell/dependency_inference.py#L154

Reply:
But anyway, this mirrors what we do for Python, for example. dep inference looks for `PythonSourceField`, and `PythonTestSourceField` extends `PythonSourceField` and so on.

Reply:
Oh wait, this is `DependenciesField`, why do we even care about that? 

I'll make this be like Python, one way or the other...

Reply:
OK, I just twisted my brain into knots for no reason. Now, post my change, this is like the Python implementation. What I meant to say above is that `PythonTestsDependenciesField` extends `PythonDependenciesField`, and python dep inference relies on this. So now shell is the same.